schema/index.go

```diff
@@ -2,12 +2,15 @@ package schema
 
 import (
 	"context"
+	"fmt"
 
 	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
+var supportedCountries = []string{"Taiwan", "US", "Iceland"}
```

Comment:
I have updated this to 
This is  CdsTaiwan, CdsUS ... etc in store package but I will move this to schema in confirm_test